src/back/deps/svetit/shared/parse/request.cpp

```diff
@@ -32,4 +32,17 @@ boost::uuids::uuid parseUUID(const server::http::HttpRequest& req, const std::st
 	return {};
 }
 
+bool parseBool(const server::http::HttpRequest& req, const std::string& key)
+{
+	try {
+		// is it right way of string to bool transformation?
+		const bool value = (strcasecmp("true",req.GetArg(key).c_str()) == 0);
+		return value;
+	} catch(const std::exception& e) {
+		const auto msg = fmt::format("Param {} parse err: {}", key, e.what());
+		throw errors::BadRequest400(msg);
+	}
+	return 0;
+}
```

Comment:
Давай может попроще?
```suggestion
bool parseBool(const server::http::HttpRequest& req, const std::string& key)
{
	try {
		const auto value = utils::text::ToLower(req.GetArg(key));
		return value == "true" || value == "1";
	} catch(const std::exception& e) {
		const auto msg = fmt::format("Param {} parse err: {}", key, e.what());
		throw errors::BadRequest400(msg);
	}
	return false;
}
```

Нужно добавить инклуд
```cpp
#include <userver/utils/text.hpp>
```

Reply:
Поскольку keepDeleted убираем, то данная функция не нужна. Оставить её на будущее тем не менее?

Reply:
Нет, если не используется, давай уберём.